docs/using_unidist/unidist_on_mpi.rst

```diff
@@ -110,13 +110,29 @@ When initializing unidist this execution model gets transformed to Controller/Wo
 Unidist on MPI cluster
 """"""""""""""""""""""
 
-In order to run unidist on MPI in a cluster, there are two options.
+Regardless of the chosen usage model (SPMD model or Controller/Worker model), there are two options for running on a cluster
 
-Controller/Worker model
------------------------
+Running with `mpiexec` command
+------------------------------
+
+This option is the most preferred and customizable.
+
+Running is almost the same as in a single node, but you should use the appropriate parameter for "mpiexec". This parameter differs depending on the mpi implementation used.
+
+For Intel MPI or MPICH: `-hosts host1,host2`. You also can see 
```

Comment:
```suggestion
* For Intel MPI or MPICH: ``-hosts host1,host2``. You can also see 
```
For MPICH it is `-host`, isn't it? https://www.mpich.org/static/docs/v3.4.x/www1/mpiexec.html#:~:text=processes%20to%20use-,%2Dhost%20%3Chostname%3E,-Name%20of%20host